File: core/php/ExchangeBase.php
<?php
class ExchangeBase{
  static function getSymbols($codes,$callback=null){}
  static function getTickers($symbols,$callback=null){}
  static function getCandles($symbol,$period=3600,$from=0,$count=100,$callback=null){}
  static function sendOrder($order){}
  static function findOrder($params){}
  static function getNamedPeriodOf($period){}
  static function getNamedExecutionModeOf($execution){}
}
?>

File: core/py/ExchangeBase.py
class ExchangeBase:
    @staticmethod
    def getSymbols(codes,callback=None):
        pass
    @staticmethod
    def getCandles(symbol,from=0,count=100,callback=None):
        pass
    @staticmethod
    def getTickers(symbols,callback=None):
        pass
    @staticmethod
    def sendOrder(order):
        pass
    @staticmethod
    def findOrder(params):
        pass
    @staticmethod
    def getNamedPeriodOf(period):
        pass
    @staticmethod
    def getNamedExecutionModeOf(execution):
        pass
